app/src/main/java/com/asfoundation/wallet/topup/TopUpFragment.kt

```diff
@@ -149,7 +164,8 @@ class TopUpFragment : DaggerFragment(), TopUpFragmentView {
     hideKeyboard()
```

Comment:
Keyboard should be shown and edit text with focus so the default values are visible immediately.